src/data/hassio/supervisor.ts

```diff
@@ -226,6 +226,25 @@ export const fetchHassioLogsFollow = async (
     signal
   );
 
+export const fetchHassioLogsFollowSkip = async (
```

Comment:
Why adding a new command instead of adding an option param to the existing `fetchHassioLogsFollow` command?

Reply:
I did it because the skip is more complex and needs the cursor and skip lines as addition parameters. But if you want I can rewrite `fetchHassioLogsFollow`